### ISSUE TITLE
Mutators appear under toolbox

### ISSUE THREAD

**Issue by NeilFraser** [google/blockly#2277]
Go to playground, choose simple mode, paste the following XML and import it:
```
<xml xmlns="http://www.w3.org/1999/xhtml">
  <block type="controls_if" x="20" y="100"></block>
</xml>
```

Open the mutator on the 'if' block.  The original behaviour (as seen at https://blockly-demo.appspot.com/static/tests/playground.html?dir=ltr&toolbox=simple&side=start) is this:
![screen shot 2019-02-08 at 11 28 43](https://user-images.githubusercontent.com/250480/52501543-708e3480-2b95-11e9-8552-72963734c337.png)
The new behaviour (as seen on 'develop') is this:
![screen shot 2019-02-08 at 11 28 29](https://user-images.githubusercontent.com/250480/52501572-800d7d80-2b95-11e9-9ba1-fdc950a41a30.png)

This regression was spotted when attempting to update Blockly Games:
![screen shot 2019-02-08 at 11 24 46](https://user-images.githubusercontent.com/250480/52501634-a0d5d300-2b95-11e9-8f0b-38ea43a8b7ff.png)

**Comment by BeksOmega** [comment]
This one's mine.

**Comment by NeilFraser** [comment]
Awesome, thanks @BeksOmega!  This allows Blockly Games to be pushed now (first public update in 9 months).

I'll leave this issue open since I heard rumours that in addition to the fixing the location, there's also interest in fixing the Z-order.

**Comment by BeksOmega** [comment]
Closed by #2279
